src/frameworks/angular.json

```diff
@@ -16,6 +16,11 @@
     "command": "ng build --prod",
     "directory": "dist/"
   },
+  "logo": {
+    "default": "https://framework-info.netlify.app/logos/angular/default.svg",
+    "light": "https://framework-info.netlify.app/logos/angular/default.svg",
+    "dark": "https://framework-info.netlify.app/logos/angular/default.svg"
```

Comment:
[dust] Do we want to only support the production domain? We could add the deploy preview if necessary (but maybe too much extra work for this and unlikely to be needed?)

Reply:
As an MVP I was just going to add the production domain since it made for slightly less testing on my part, but I can take a look at making this change while I wait for reviews 😄

Reply:
Have added it to the base branch of this PR in [this commit](https://github.com/netlify/framework-info/pull/797/commits/20f8e7c410d81229279d4fe45bce6b634ab9070f), so assuming that's a good approach I can update the paths here as well to support non-production domains